### PR TITLE
feat: add example configuration with drone.

### PR DESCRIPTION
**Pull request by appleboy**
https://github.com/drone/drone

### CODE REVIEW

File: drone/Caddyfile [hunk 1]
```diff
@@ -0,0 +1,8 @@
+example.com {
+  proxy / localhost:8000 {
+    websocket
+    header_upstream X-Forwarded-Proto {scheme}
+    header_upstream X-Forwarded-For {host}
+    header_upstream Host {host}
+  }
+}
```
Comment (mholt) [inline drone/Caddyfile]: Isn't this the same as using `transparent`? (the header_upstreams)

Comment (appleboy) [inline drone/Caddyfile]: @mholt Good catch. Updated. Thanks.

**Comment by mholt** [comment]
Does Drone require Caddy 0.9.5's timeouts to be disabled? (i.e. for requests longer than ~10s)

**Comment by appleboy** [comment]
@mholt Drone can't work with caddy 0.9.5 version. It is working on caddy 0.9.4 version. I already notice this information to the author of drone. I will try your solution on 0.9.5. Please wait a moment.

**Comment by appleboy** [comment]
Working on 0.9.4 version:

```
example.com {
  proxy / localhost:8000 {
    websocket
    transparent
  }
}
```

Can't work with `0.9.5`

```
example.com {
  timeouts {
    read 0
  }
  proxy / localhost:8000 {
    websocket
    transparent
  }
}
```

**Comment by appleboy** [comment]
@mholt It is working on 0.9.5 as following:

```diff
example.com {
+ timeouts none
  proxy / localhost:8000 {
    websocket
    transparent
  }
}
```

**Comment by appleboy** [comment]
@mholt Done. Please review again.

**Comment by mholt** [comment]
Thank you!

**Comment by tjakobsson** [comment]
Adding "timeouts none" also helped when proxying Rancher!

**Comment by mholt** [comment]
@tjakobsson Cool, want to submit that change?

**Comment by tjakobsson** [comment]
I don't know what you mean by that. It was only a configuration change (which wasn't needed in 0.9.4 though). The full configuration is as follows, starting caddy with --http2=false:

```
example.com {
  tls tjakobsson@example.com

  timeouts none

  proxy / http://localhost:8080 {
    transparent
    websocket
  }
}
```

**Comment by mholt** [comment]
@tjakobsson I mean update the configuration example in this repository :)

**Comment by appleboy** [comment]
@tjakobsson What is different from this repository and your example?

**Pull request merged**